packages/base-cards/string/edit.js

```diff
@@ -3,20 +3,21 @@ import { precompileTemplate } from '@ember/template-compilation';
 import templateOnlyComponent from '@ember/component/template-only';
 import { on as realOn } from '@ember/modifier';
 import { helper } from '@ember/component/helper';
+import BoxelInput from '@cardstack/boxel/components/boxel/input';
 
 // https://github.com/ember-cli/babel-plugin-htmlbars-inline-precompile/issues/379
 let on = realOn;
 
 const getTargetValue = helper(([action] /*, hash*/) => {
-  return function (event) {
-    action(event.target.value);
+  return function (value) {
+    action(value);
   };
 });
 
 export default setComponentTemplate(
-  precompileTemplate('<input type="text" value="{{@model}}" {{on "input" (getTargetValue @set)}} ...attributes />', {
+  precompileTemplate('<BoxelInput type="text" @value="{{@model}}" @onInput={{getTargetValue @set}} ...attributes />', {
```

Comment:
It looks like we don't need a helper here at all anymore and `@onInput={{getTargetValue @set}}` can become `@onInput={{@set}}`